health-services/project-factory/src/server/utils/projectCampaignUtils.ts

```
@@ -0,0 +1,565 @@
+                        target.targetNo = newTargets[target.beneficiaryType];
+                    }
+                });
+            }
+        });
+
+        // Update only the relevant campaign projects
+        const campaignProjectsToUpdate = campaignProjects
+            .filter((campaignProject: any) => projectIds.includes(campaignProject.projectId))
+            .map((campaignProject: any) => {
+                campaignProject.additionalDetails.targets = boundaryCodesForTargetUpdateMappings[campaignProject.boundaryCode]?.targets;
+                delete campaignProject.additionalDetails.newTargets;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)